mirgecom/mechanisms/__init__.py

```diff
@@ -87,6 +87,11 @@ def get_mechanism_cti(mechanism_name: str) -> str:
 
 def get_mechanism_input(mechanism_name: str) -> str:
     """Get the contents of a mechanism YAML input file."""
+    if mechanism_name == "uiuc":
+        from warnings import warn
+        warn("The uiuc mechanism was updated in Q3 2023. "
+             "The previous mechanism is now called uiuc_xpacc.yaml.",
+             stacklevel=2)
```

Comment:
Let's delete the old mechanism and the symbolic link, and just reset the mech name to use the right one.  The warning should be enough to spur folks (all 3 or 4 of us) into getting it changed. 

```suggestion
    if mechanism_name == "uiuc":
        from warnings import warn
        warn("The uiuc mechanism was updated in Q3 2023. "
             "The previous mechanism is now called uiuc_xpacc.yaml.",
             stacklevel=2)
        mechanism_name="uiuc_xpacc.yaml"
```